warehouse/templates/manage/releases.html

```diff
@@ -121,7 +126,6 @@
               </a>
             </li>
             {% endif %}
-            {% endif %}
           </ul>
```

Comment:
I don't know what you mean by the `endif` moving :sweat_smile: I added an `if .. else` statement here because the idea is that `(un)yaking` only make sense on published releases, as it's a user-facing operation, and draft releases are invisible to regular users of the packages.

And yes, you should be able to delete draft releases or the files within them.

Reply:
(I read the PR commit by commit, and on the commit where I commented, this endif was moved from ~10 lines above to here.)